src/pages/events/create-events.md

```diff
@@ -41,17 +45,85 @@ Field | Description
 
 ### Configure event subscription fields
 
-The **Event Subscription Fields** configuration panel allows you to define the fields of the event payload to transmit from Commerce. The name provides the path to the field in the event payload. See [Register events](./module-development.md#register-events) for details on specifying event field names.
+The **Event Subscription Fields** configuration panel allows you to define the fields of the event payload to transmit from Commerce. The name provides the path to the field in the event payload.
+
+<SampleEvent />
+
+<InlineAlert variant="info" slots="text" />
+
+If you do not specify any fields, Commerce sends the entire event payload. Adobe recommends sending a limited number of fields per event. If you send all fields, you increase the risk of including sensitive or PCI compliance data in the event. In addition, specifying only the fields that are applicable to your business case is recommended for optimal performance and cost effectiveness.
+
+The following example shows how three fields can be configured for the `observer.catalog_product_save_after` event:
+
+```yaml
+Name: entity_id
+Name: sku
+Name: is_new
+```
+
+The contents of an `observer.catalog_product_save_after` event are similar to the following:
+
+```json
+{
+    "value": {
+    "entity_id": "3",
+    "sku": "test2",
+    "is_new": "0"
+  }
+}
```

Comment:
```suggestion
{
    "entity_id": "3",
    "sku": "test2",
    "is_new": "0"
}
```

I think this is an error that was already in the docs, but there would not be a value key